src/molecules/Tabs/Tabs.native.js

```diff
@@ -0,0 +1,125 @@
+import React, { useCallback, useEffect } from 'react';
+import { Dimensions } from 'react-native';
+import PropTypes from 'prop-types';
+import { TabView } from 'react-native-tab-view';
+import { getIconNames } from '../../_helpers/icon';
+import isDefined from '../../_helpers/isDefined';
+import TabBar from './TabBar';
+
+const initialLayout = { width: Dimensions.get('window').width };
+
+const Tab = () => {
```

Comment:
`Tab` should be in a different file `Tab.native.js`

Reply:
I'm not sure if that makes sense because Tab as a component does nothing really. It's just for the API. It only accepts some props which we then use to render the actual TabView. Should I still extract it out in a different file?

Reply:
Yes same applies to `TextAccessoryIcon` right? Different files doesn't mean they should be used directly

Reply:
Okay also why are we returning Tab as null?

Reply:
That's the right question 😂

Reply:
Let's get on a quick call?